scipy/stats/_continuous_distns.py

```diff
@@ -6921,7 +6921,10 @@ def _entropy(self, c, d):
         return 0.5 * (1.0-d+c) / (1.0+d-c) + np.log(0.5 * (1.0+d-c))
 
 
-trapz = trapz_gen(a=0.0, b=1.0, name="trapz")
+trapezoid = trapezoid_gen(a=0.0, b=1.0, name="trapezoid")
+# Note: alias kept for backwards compatibility. Rename was done
+# because trapz is a slur in colloquial English (see gh-12924).
+trapz = trapezoid_gen(a=0.0, b=1.0, name="trapz")
```

Comment:
This appears insufficient for the doc system to correctly realize the shape parameters of trapz and generates name errors for 'c', 'd'  .

Reply:
Ah, I see what is going on there. I should have just replaced the `trapz.__doc__` docstring to mention it's an alias and refer to `trapezoid`.